backend/src/apiserver/config/config.json

```diff
@@ -10,6 +10,8 @@
     "SecretAccessKey": "minio123",
     "BucketName": "mlpipeline"
   },
+  "KFAM_SERVICE_HOST": "profiles-kfam.kubeflow",
```

Comment:
The pod environment variable should have this through the format of IP address.

Reply:
Without this config, the unit tests will fail when initializing the kfam client.

Reply:
the unit test should use the fake client instead of relying on the prod config.

Reply:
Right. I added the fake, I will remove it now.